lib/routes/ehentai/search.ts

```diff
@@ -25,15 +25,16 @@ async function handler(ctx) {
     const page = ctx.req.param('page');
     let params = ctx.req.param('params');
     const routeParams = new URLSearchParams(ctx.req.param('routeParams'));
-    const bittorrent = routeParams.get('bittorrent') || false;
-    const embed_thumb = routeParams.get('embed_thumb') || false;
+    const bittorrent = routeParams.get('bittorrent') === 'true';
+    const embed_thumb = routeParams.get('embed_thumb') === 'true';
+    const my_tags = routeParams.get('my_tags') === 'true';
```

Comment:
Do not change the default value.
https://github.com/DIYgod/RSSHub/blob/e94993264320291056469bb4ee6dc418d963721e/lib/routes/ehentai/namespace.ts#L9-L11